src/platforms/apple/common/performance/instrumentation/swiftui-instrumentation.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: SwiftUI Instrumentation
+sidebar_order: 12
+description: "Learn how to monitor the performance of your SwiftUI views."
+---
+
+<Note>
+    This is an experimental feature.
+</Note>
```

Comment:
Please consider replacing this with `<Include name="alpha-note.mdx" />` or `<Include name="beta-note.mdx" />` or add an extra reusable include like the previous ones.